approved-books.json

```diff
@@ -1,3 +1,354 @@
 [
-  {"name": "example-book", "server": "qa.cnx.org", "collection_id": "col12345", "version": "0.1.0", "style": "example" }
-]
+  {
+    "collection_id": "col11762",
+    "version": "14.1",
```

Comment:
General question for all of these version values: Just to confirm, are they the UUID@version or the collection_id@version? The pipeline will end up consuming these as though they're the collection_id@version variant, but maybe we need to revisit our naming conventions here if we're including both UUID and collection_id in the feed to avoid confusion?

Reply:
If we prepend `1.` onto all these versions, this would resolve the ambiguity for neb, right?